src/openforms/authentication/views.py

```diff
@@ -425,3 +431,25 @@ def form_valid(self, form):
 
         assert self.cleaned_next_url
         return HttpResponseRedirect(self.cleaned_next_url)
+
+
+class AuthPluginLogoutView(UserPassesTestMixin, View):
+    http_method_names = ["post"]
+    raise_exception = True
+
+    def test_func(self):
+        return FORM_AUTH_SESSION_KEY in self.request.session
```

Comment:
Because of the `raise_exception = True` it will raise a `PermissionDenied` which will give a 403 (the `request.user.is_authenticated` is always false unless it is a form designer who is loggedin in the admin).

Since one in theory should not see the logout button until they are authenticated, I don't think they should be redirected anywhere if they try to make calls to the logout endpoint? :thinking:

Reply:
> I don't think they should be redirected anywhere if they try to make calls to the logout endpoint?

That's exactly the wrong behaviour that sparked the question in my head. :slightly_smiling_face: